hooks/pg_dir_hooks.py

```diff
@@ -117,13 +124,17 @@ def config_changed():
     if charm_config.changed('plumgrid-license-key'):
         if is_leader() and post_pg_license():
             log("PLUMgrid License Posted")
+        # Post PG license to Sol-API
+        sapi_post_license()
     if charm_config.changed('fabric-interfaces'):
         if not fabric_interface_changed():
             log("Fabric interface already set")
         else:
             stop_pg()
     if charm_config.changed('plumgrid-virtual-ip'):
         CONFIGS.write_all()
+        for rid in relation_ids('plumgrid'):
+            plumgrid_joined(rid)
```

Comment:
I believe plumgrid_joined only sets the opsvm IP in the relation. Why would you need it here?